sources/packages/backend/libs/sims-db/src/entities/dynamic-form-configuration.model.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { BaseModel, DynamicFormType, OfferingIntensity, ProgramYear } from ".";
+import { ColumnNames, TableNames } from "../constant";
+
+/**
+ * Dynamic form configurations based on form type.
+ */
+@Entity({ name: TableNames.DynamicFormConfigurations })
+export class DynamicFormConfiguration extends BaseModel {
+  /**
+   * Auto-generated sequential primary key column.
+   */
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  /**
+   * Form type which is dynamically configured.
+   */
+  @Column({
+    name: "form_type",
```

Comment:
Minor, should we explicitly mention the string type here for better clarity, even if not required?